app/views/catalog/_facet_pagination.html.erb

```diff
@@ -1,3 +1,13 @@
+    <% if @facet.index_pagination && params["facet.sort"] == 'index' %>
+      <div class="alpha-filter">
+        <%= t('blacklight.search.facets.filter') %>:
```

Comment:
I /think/ the `:` should be part of the i18n label. We should also wrap the label and content in some element (`span` or `div` or otherwise?) so they can be styled downstream (or, for RTL languages, swapped around)